core/common.go

```diff
@@ -0,0 +1,24 @@
+package core
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"io"
+)
+
+// RandomString and NewToken come from Boulder core/util.go
```

Comment:
AFAICT RandomString and NewToken are only used in wfe.go? So they should probably stay there. Creating a common.go is risky new ground. :-) I would also support creating a token.go in the wfe directory, if that's preferable.